### DISCUSSION TITLE
Does Player SDK support language internationalization?

### DISCUSSION THREAD

**Discussion by WWPyinji** [vimeo/player.js#750] [Q&A]
I want to modify the language of each button and prompt of the vimeo player，what should I do?

**Comment by fisherinnovation** [comment]
The language used in the Vimeo Player is selected based of either your configured language on Vimeo.com or your browsers preferred language pending it is supported by the Vimeo Player. There is currently no method to change this manually.

More info on supported languages can be found at https://vimeo.zendesk.com/hc/en-us/articles/360001851407-Languages-supported-on-Vimeo

I will also bring back your request to the product team.
